pwa-devdocs/src/pagebuilder/custom-components/setup-component/index.md

```diff
@@ -0,0 +1,130 @@
+
+![Component file structure](PageBuilderFilesSetup.png)
+
+Starter code for the `ExampleQuote` component files is provided as follows.
+
+### configAggregator.js
+
+```js
+export default (node, props) => {
+    return {
+        // Retrieve properties from node here
+    };
+};
+```
+
+### exampleQuote.css
+
+```css
+/* class for testing setup */
+.testClass {
+    line-height: 1.4em;
+    font-size: 1em;
+    color: #0066cc;
+}
+```
+
+### exampleQuote.js
+
+```jsx
+import React from 'react';
+import { mergeClasses } from '../../../../../classify';
+import defaultClasses from './exampleQuote.css';
+import { shape, string } from 'prop-types';
+
+const classes = mergeClasses(defaultClasses, props.classes);
+
+// Component for testing setup
+const ExampleQuote = props => {
+    return (
+        <div className={classes.testClass}>
+            <div>Content Type: {props.contentType}</div>
+            <div>Appearance: {props.appearance}</div>
+        </div>
+    );
+};
+
+ExampleQuote.propTypes = {
+    classes: shape({
+        testClass: string
+    }),
+    contentType: string,
+    appearance: string
+}
+
+export default ExampleQuote;
+```
+
+### index.js
+
+```js
+export { default } from './exampleQuote';
+```
+
+## Step 2: Configure component
+
+ Configuring your component is about connecting your component to the Page Builder component framework. The framework provides a static config object (in `config.js`) that acts as a kind of registry of content types supported within PWA Studio. To include support for your custom content type component, you'll need to add an entry to the `config.js`:
+
+![Component config file](PageBuilderConfigFile.png)
+
+Open the `config.js` file and `import` your `configAggregator`. The import statement for our ExampleQuote is shown here:
+
+```js
+import exampleQuoteConfigAggregator from './ContentTypes/ExampleQuote/configAggregator';
+```
+
+Add an entry that references your `configAggregator` and `component`, using the name of your content type as the `key`. For our `ExampleQuote`, the name of the Quote content type is `example_quote`, so the entry for our Quote component (`ExampleQuote`) in the config file looks like this:
+
+```js
+example_quote: {
+    configAggregator: exampleQuoteConfigAggregator,
+    component: React.lazy(() => import('./ContentTypes/ExampleQuote'))
+},
+```
+
+### A note about lazy loading components
+
+When building PWAs, performance is key. That's why we recommend loading less critical components using `React.lazy`. This will result in a very slight delay in that content rendering but will ensure we don't bloat the bundle size for the store. For more information, see [React lazy loading].
```

Comment:
```suggestion
When building PWAs, performance is key. That's why we recommend loading less critical components using `React.lazy`. This will result in a very slight delay in that content rendering but will ensure we don't bloat the bundle size for the store. For more information, see [React lazy loading][].
```